int32[] detected_corners
int32 sense
int32 view
int32[] detected_ids
